Header header
float64 ex
float64 ey
float64 dx
float64 dy
float64 avg_ex
float64 avg_ey
float64 avg_dx
float64 avg_dys